src/Control/Controller.php

```diff
@@ -628,12 +618,8 @@ public function popCurrent()
 
     /**
      * Redirect to the given URL.
-     *
-     * @param string $url
-     * @param int $code
-     * @return HTTPResponse
      */
-    public function redirect($url, $code = 302)
+    public function redirect(string $url, int $code = 302): HTTPResponse
```

Comment:
This method has a `return null;` statement in it

Reply:
Updated